write-fonts/src/tables/glyf.rs

```diff
@@ -250,8 +250,19 @@ impl RepeatableFlag {
     ) -> impl Iterator<Item = RepeatableFlag> {
         let mut iter = flags.into_iter();
         let mut prev = None;
+        // if a flag repeats exactly once, then there is no (space) cost difference
+        // between using a repeat flag followed by a value of '1' and just repeating
+        // the flag (without setting the repeat bit). Our approach naturally suggests
+        // the first approach, but fontmake takes the second, so we add an extra
+        // step where if we see a case where there's a single repeat, we split it into
+        // two separate non-repeating flags.
```

Comment:
I would view matching existing as the default choice absent a reason to do otherwise. We know fontmake emits fonts that works, we know we want our compiler built on this code to produce the same output as fontmake other than well-understood differences. So ... what is our reason to do other than what fontmake does?

Reply:
Hm. I misread I think. So you *do* match fontmake and the key point is we're choosing repetition specifically to match fontmake. Perhaps we could say that more directly?

```
        // if there are two consecutive identical flags, then there is no (space) cost difference
        // between 1) using a repeat flag followed by a value of '1' and 2) repeating
        // the flag without setting the repeat bit. fontmake takes option 2) so we match
        // that here.
```

Feel free to ignore if you feel this no better than the original explanation.